metricflow/test/snapshots/test_conversion_metrics_to_sql.py/SqlQueryPlan/test_conversion_count_with_no_group_by__plan0.xml

```diff
@@ -24,102 +30,207 @@
             <!-- where = None -->
             <!-- distinct = False -->
             <SqlSelectStatementNode>
-                <!-- description = Aggregate Measures -->
-                <!-- node_id = ss_11 -->
-                <!-- col0 =                                                                                                          -->
-                <!--   SqlSelectColumn(expr=SqlAggregateFunctionExpression(node_id=fnc_0, sql_function=SUM), column_alias='visits')  -->
+                <!-- description = 'Aggregate Measures' -->
+                <!-- node_id = NodeId(id_str='ss_11') -->
+                <!-- col0 =                                                                     -->
+                <!--   SqlSelectColumn(                                                         -->
+                <!--     expr=SqlAggregateFunctionExpression(node_id=fnc_0, sql_function=SUM),  -->
+                <!--     column_alias='visits',                                                 -->
+                <!--   )                                                                        -->
                 <!-- from_source = SqlSelectStatementNode(node_id=ss_10) -->
                 <!-- where = None -->
                 <!-- distinct = False -->
                 <SqlSelectStatementNode>
-                    <!-- description = Pass Only Elements: ['visits',] -->
-                    <!-- node_id = ss_10 -->
+                    <!-- description = "Pass Only Elements: ['visits',]" -->
```

Comment:
Should we default to double quotes for these descriptions for better consistency? Not a strong opinion here.

Reply:
Do you mean how it uses `"` in some cases but `'` in others? It seems that's the behavior of `repr()` / `pprint.pformat()` if it's a string or a string containing a quote.